### PR TITLE
Report the correct error when we cannot remove a network

### PR DESCRIPTION
**Pull request by CameronDiver**
Change-type: patch
Signed-off-by: Cameron Diver <cameron@balena.io>

### CODE REVIEW

File: src/compose/network.ts [hunk 1]
```diff
@@ -193,7 +193,7 @@ export class Network {
 				.getNetwork(Network.generateDockerName(this.appId, this.name))
 				.remove(),
 		).tapCatch(error => {
-			this.logger.logSystemEvent(logTypes.createNetworkError, {
+			this.logger.logSystemEvent(logTypes.removeNetworkError, {
 				network: { name: this.name, appId: this.appId },
```
Comment (richbayliss) [inline src/compose/network.ts]: is this right?

Comment (CameronDiver) [reply src/compose/network.ts]: How do you mean?

Comment (richbayliss) [reply src/compose/network.ts]: Ok - reason for the false flag; the PR title is "... cannot remove a volume" and this is all about networks...

Comment (CameronDiver) [reply src/compose/network.ts]: Ah yeah, accidentally put volume instead of network in the commit message - changed now. Nice catch!

**Pull request merged**